File: Bhaskara.c
#include <stdio.h>
#include <math.h>

double calculaX1(double a, double b, double c) {
   double delta = (b*b)-4*(a)*(c);
   return (-b + sqrt(delta)) / (2*a);
}

double calculaX2(double a, double b, double c) {
   double delta = (b*b)-4*(a)*(c);
   return (-b - sqrt(delta)) / (2*a);
}

int main(int argc, char** argv)  {
   double  a, b, c;
     
   printf("X1 = %lf", calculaX1(2, -5, -7));
   printf("X2 = %lf", calculaX2(2, -5, -7));
   return 0;
}

File: Bhaskara.py
import math

class Bhaskara:
    def __init__(self, a, b, c):
        self.a = a
        self.b = b
        self.c = c

    def calcula_x1(self):
        delta = (self.b **2 -4 *  self.a * self.c) 
        return (-self.b + math.sqrt(delta)) / (2 * self.a)

    def calcula_x2(self):
        delta = (self.b **2 -4 * self.a  * self.c)
        return (-self.b - math.sqrt(delta)) / (2 * self.a)

# Executando o código
bascara = Bhaskara(2, -5, -7)
print(bascara.calcula_x1())
print(bascara.calcula_x2())